docs/architecture/devops-for-aspnet-developers/toc.yml

```diff
@@ -16,9 +16,9 @@
     items:
     - name: Comparison of GHA and Azure Pipelines
       href: mock.md
-    - name: Basic Build Workflow 
+    - name: Basic build workflow 
       href: actions-build.md
     - name: Adding deployment steps
-      href: mock.md
+      href: actions-deploy.md
     - name: Securing .NET code with GitHub Actions and CodeQL
-      href: mock.md
\ No newline at end of file
+      href: mock.md
```

Comment:
Is _mock.md_ a file you plan to create? I ask because this link is broken right now.

Reply:
Yes, the labs are coming in the next PR